experimental/babel-preset-env/README.md

```diff
@@ -181,7 +181,7 @@ For more information on setting options for a preset, refer to the [plugin/prese
 
 Takes an object of environment versions to support.
 
-Each target environment takes a number or a string (we recommend using a string when specifying minor versions like `node: "6.10"`).
+Each target environment takes a number or a string (we recommend using a string when specifying minor versions like `node: "6.10"`). You can also specify `tp` (technology preview) version for Safari.
```

Comment:
I don't think we need a whole line of documentation just for safari tp? or I would just name the value `safari tp`? below

Reply:
@hzoo According to the compat-table it's pretty static value.
All chrome canary releases are labeled under number value. For example, chrome 64 (canary).
Same for firefox: FF 58 (Nightly).
Edge 17 (Preview).
But for Safari it's just TP. And when new TP become 11.2 for example, the new one will be labeled as tp also.